web/src/bootstrap_typeahead.ts

```diff
@@ -242,7 +243,7 @@ export class Typeahead<ItemType extends string | object> {
         } else {
             assert(!this.input_element.$element.is("[contenteditable]"));
         }
-        this.items = options.items ?? 8;
+        this.items = options.items ?? 50;
```

Comment:
Let's define this as a constant and use that everywhere else so that it is easier to adjust later.

Reply:
Done!